sources/constraint-declaration-validation.asciidoc

```diff
@@ -816,16 +816,13 @@ private Map<String, @NotEmpty List<@ValidAddress Address>> addressesByType;
 
 In such case multiple value extractors will be invoked.
 
-[tck-testable]
```

Comment:
It's the explanation of an example. We have other tests covering this and we usually don't implement all the examples as TCK tests.